docs/management/gcache-write-set-cache-encryption.md

```diff
@@ -0,0 +1,96 @@
+# GCache and Write-Set cache encryption
+
+The feature is a tech preview. Before using this feature in production, we recommend that you test restoring production from physical backups in your environment, and also use the alternative backup method for redundancy.
+
+## GCache encryption Master Key
+
+GCache encrypts with a File Key. Creating the galera.gcache file or a full reset generates the File Key. The File Key is encrypted with the Master Key. 
+
+GCache has a a RingBuffer on-disk file to manage write-sets. The file is pre-created when the server starts. This file contains a 1KB preamble to hold key-value pairs. The Master Key id and the encrypted File Key are stored in the GCache Ring preamble. The preamble is not encrypted because this section does not store sensitive data. 
```

Comment:
```suggestion
GCache has a RingBuffer on-disk file to manage write-sets. The file is pre-created when the server starts. This file contains a 1KB preamble to hold key-value pairs. The Master Key id and the encrypted File Key are stored in the GCache Ring preamble. The preamble is not encrypted because this section does not store sensitive data. 
```

Reply:
removed the extra "a"